docker-compose.yml

```diff
@@ -1,54 +1,48 @@
-version: "3"
+version: "3.4"
 
 services:
-
-  db:
+  postgresql:
     image: postgres:9.6
     env_file:
         - env.d/development/postgresql
     ports:
         - "5440:5432"
 
+  mysql:
+    image: mysql:5.6
+    env_file:
+      - env.d/development/mysql
+    ports:
+      - "3316:3306"
+    command: mysqld --character-set-server=utf8 --collation-server=utf8_general_ci
```

Comment:
IMO this is a regression... running 2 db containers on your system seems like a burden.

Reply:
Only one will! Look at the app service dependencies!

Reply:
all the magic also happen in the makefile. By running only the `app` container.

Reply:
...and it's dynamic dependencies!

![](https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif)